lib/dor/services/client/object.rb

```diff
@@ -38,6 +38,10 @@ def version
           @version ||= ObjectVersion.new(**parent_params)
         end
 
+        def user_version
+          @user_version ||= UserVersion.new(**parent_params)
```

Comment:
This looks like it might be missing test coverage in `spec/dor/services/client/object_spec.rb`

Reply:
Added. My least favorite kind of test coverage.

Reply:
Yeah, I could go either way on that batch of tests. On one hand, I don't think we need to test whether delegation works. On the other, I'd want to know if we somehow broke the delegation or got it wrong. Maybe that's not a thing worth being concerned about (i.e., because costs may outweigh benefits)?